### PR TITLE
[fix][subscribers] Regenerate metro status dict

### PR DESCRIPTION
**Pull request by jidicula**
**Why this change was necessary**
The metro status dictionary was not being regenerated correctly,
leading to type errors. There was also an incorrect comparison being
made in the logical conditions that would trigger a status update
being sent and saved.

**What this change does**
Refactors globals for default status messages and metro
colours. Refactors metro status dict and fixes conditional comparison
for triggering a status being posted in the server and updated in the dict.

**Any side-effects?**
None.

**Additional context/notes/links**
Note that this could not be adequately tested as metro outages are
infrequent.

Thanks to @davidlougheed for the refactor work.

Resolves: #221 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: cogs/subscribers.py [hunk 1]
```diff
@@ -122,11 +135,11 @@ def check_status(line_number, response):
             response = requests.request("GET", METRO_STATUS_API)
             for line_status in metro_status.items():
                 line_number = line_status[0]
-                cached_status = line_status[1][0]
-                line_colour = line_status[1][1]
+                cached_status = line_status[1]
+                line_colour = METRO_COLOURS[line_number]
```
Comment (davidlougheed) [inline cogs/subscribers.py]: this variable is probably unnecessary since it's just used once

File: cogs/subscribers.py [hunk 2]
```diff
@@ -122,11 +135,11 @@ def check_status(line_number, response):
             response = requests.request("GET", METRO_STATUS_API)
```
Comment (davidlougheed) [inline cogs/subscribers.py]: use `requests.get` instead

File: cogs/subscribers.py [hunk 3]
```diff
@@ -122,11 +135,11 @@ def check_status(line_number, response):
...
             for line_status in metro_status.items():
                 line_number = line_status[0]
-                cached_status = line_status[1][0]
-                line_colour = line_status[1][1]
+                cached_status = line_status[1]
+                line_colour = METRO_COLOURS[line_number]
                 current_status = check_status(line_number, response)
```
Comment (davidlougheed) [inline cogs/subscribers.py]: expand out tuple so it makes more sense and lines up with the other `_status` variable, e.g.:
```python
line_name, current_status = check_status(line_number, response)
```

File: cogs/subscribers.py (hunk 2 above)
Comment (jidicula) [reply cogs/subscribers.py]: Resolved by d13c353.

File: cogs/subscribers.py (hunk 1 above)
Comment (jidicula) [reply cogs/subscribers.py]: Resolved by d13c353.

File: cogs/subscribers.py (hunk 3 above)
Comment (jidicula) [reply cogs/subscribers.py]: Resolved by d13c353.

**Pull request merged**